### ISSUE TITLE
Improve our Protractor tests and the self-documentation

### ISSUE THREAD

**Issue by adiherzog** [scenarioo/scenarioo#511]
Our self-docu should be a good example of a Scenarioo documentation. Therefore we should improve the structure / naming / labels / details, etc. of it to match our quality standards.

**Comment by bruderol** [comment]
:+1: Yes!

**Comment by danielsuter** [comment]
After discussion:
* improve labels (better naming, more labels)
* check if we are already using the lastest scenarioo-js version (clarify with @bruderol  first)
* sometimes the screenshot is taken too late or early (e.g. pop ups not visible)
* verify use case and scenarioo structure - does it generate a proper documentation?

**Comment by bruderol** [comment]
@danielsuter Yes, Scenarioo is using the latest ScenariooJS version. I updated it. Let me know if you need any help with this. I have many experience in this from my current project.
